tests/test_fixity.py

```diff
@@ -18,6 +18,7 @@
 STORAGE_SERVICE_USER = "test"
 STORAGE_SERVICE_KEY = "test"
 REPORT_URL = "http://localhost:8003/"
+logger = logging.getLogger(__name__)
```

Comment:
Aren't these tests supposed to use the logger you're setting up in `fixity.main`?